src/Symfony/Component/Runtime/Internal/autoload_runtime.template

```diff
@@ -12,6 +12,16 @@ if (!is_object($app)) {
     throw new TypeError(sprintf('Invalid return value: callable object expected, "%s" returned from "%s".', get_debug_type($app), $_SERVER['SCRIPT_FILENAME']));
 }
 
+if (null === ($_ENV['APP_RUNTIME_MODE'] ??= $_SERVER['APP_RUNTIME_MODE'] ?? null)) {
+    if ($_ENV['FRANKENPHP_WORKER'] ?? $_SERVER['FRANKENPHP_WORKER'] ?? false) {
```

Comment:
Do you really need to have this specific check? Wouldn't that be something to put in FrankenPHP instead?

Reply:
Or in https://github.com/php-runtime/frankenphp-symfony?

Reply:
We could do that also for CLI and SAPI. The main benefit of doing it here is to not have to duplicate this code in every runtime supporting FrankenPHP worker mode.

Reply:
Setting `APP_RUNTIME_MODE` in the bridge would be a better demonstration of how this have to be configured. Otherwise we'll receive PR to add Bref or React support in this file.